src/layout/section/heroSection/index.tsx

```diff
@@ -1,123 +1,168 @@
+import styled, { css } from 'styled-components'
 import { responsiveBreakpoints, componentSizes, color, space } from '_utils/branding'
+import HeroSection, { HeroSectionProps } from './heroSection'
 
-import HeroSection from './heroSection'
+// Remove these deprecated styles once all the button content parts have been removed.
+const DEPRECATED_STYLES = css`
+  & {
+    justify-content: flex-end;
+  }
+
+  & .hero-info {
+    padding-top: 0;
+    padding-bottom: ${space.xl};
+    margin-bottom: ${space.xl};
+  }
+`
+
+const DEPRECATED_SMALL_MEDIA_STYLES = css`
+  & {
+    margin-bottom: 0;
+    height: initial;
+  }
+
+  & .hero-image {
+    position: static;
+    height: 136px;
+    background-image: ${(props: HeroSectionProps) => `url(${props.heroImageUrl})`};
+  }
+
+  & .hero-image::after {
+    content: none;
+  }
+
+  & .hero-info {
+    padding-top: ${space.xl};
+  }
+
+  & .hero-info-title {
+    color: ${color.primaryText};
+  }
+
+  & .hero-info-text {
+    color: ${color.secondaryText};
+    font-size: 16px;
+  }
+
+  & .hero-content {
+    align-items: center;
+  }
+`
 
 const StyledHeroSection = styled(HeroSection)`
   & {
     background-color: ${color.defaultBackground};
     position: relative;
     z-index: 1;
-    padding: 0;
+    height: 75vh;
+    margin-bottom: ${space.xl};
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
   }
 
   & .hero-image {
-    position: relative;
-    min-height: 136px;
+    position: absolute;
+    z-index: -1;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
     background-size: cover;
     background-position: center;
-    background-image: ${props => `url(${props.heroImageUrl})`};
+    background-image: ${props => `url(${props.heroImageUrlLarge})`};
+
+    /* Hide the shadow. */
+    overflow: hidden;
+  }
+
+  & .hero-image::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 75%;
+    display: block;
+    background: linear-gradient(
+      0deg,
+      rgba(0, 0, 0, 0.4) 72%,
+      rgba(0, 0, 0, 0.2) 85%,
+      rgba(0, 0, 0, 0) 100%
+    );
   }
 
   & .hero-info {
-    text-align: center;
-    margin-left: auto;
-    margin-right: auto;
+    width: 100%;
     max-width: ${componentSizes.largeSectionWidth};
-    padding: ${space.xl};
-    padding-top: 0;
+    padding: 0 ${space.xl};
+    padding-top: 120px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
   }
 
   & .hero-info-title-wrapper {
-    display: flex;
-    flex-direction: column;
+    text-align: center;
   }
 
-  & .hero-info-title.kirk-title {
-    min-height: auto;
-    align-items: flex-end;
-    align-self: center;
-    display: flex;
-    padding-top: ${space.xl};
-    padding-bottom: ${space.ms};
+  & .hero-info-title {
     margin: 0;
+    padding-top: ${space.m};
+    padding-bottom: ${space.m};
+    color: ${color.textWithBackground};
+    font-size: 60px;
```

Comment:
Please use font-sizing

Reply:
Actually you should use Typography components instead

Reply:
Problem is that the homepage was never fully Pixar compliant, it was copied as-is from the existing page. There's nothing in Pixar that has this size for a Title (this could be discussed with UX)

Reply:
I agree we should not have font-size that is not in the branding. However this is not the goal of this PR and @simonrelet is not introducing this, he just moved it

Reply:
Ok.. can he use the ones that already exist then.. and let only `60px` outside of scale?

Reply:
I can use the typography components instead but I still need to override some `font-size` and `line-height` values as they only have one size for both small and large screens. The font sizes in the design are not the same for both devices.

From the design (small / large devices):

- "Title": 30px / 60px
- "Text": 18px / 22px

This means:
- "Title" can be a `Display1` (30px) with an override (60px) for large device.
- "Text" can either be a:
  - `SubHeader` (22px) with an override (18px) for small device
  - `Title` (18px) with an override (22px) for large device

What do you think?

Reply:
Unfortunatelly you must override. Please, start **mobile first** that means `Title`  going up to `22px` 

I also recommend override with _styled-components_ instead of css classes

Reply:
> I also recommend override with styled-components instead of css classes
> @gabrielagabriel 

I'm not sure to understand, what are you proposing?

Reply:
```
const HeroTitle = styled(Title)`
// heroTitle styles here
`
```

Reply:
I kept the class names as there were until we agree on the way to use styled-components.